src/bots/mascot-bot.js

```diff
@@ -39,6 +46,20 @@ class MascotBot extends SlackBot {
     });
   }
 
+  setTopic(channel, topic, isPublic = true) {
```

Comment:
Okay, so private rooms are groups. Maybe rename the param to `isPublicRoom` or `isPublicChannel` then?